.github/workflows/test.yaml

```
@@ -126,8 +129,8 @@ jobs:
           mode: recreate
           create_if_not_exists: true
 
-  live-tests:
-    name: Run Live Tests
+  test-live:
+    name: live
```

Should we keep semantics here?

```suggestion
  test-e2e:
    name: e2e
```